flatdata-cpp/include/flatdata/MultiVector.h

```diff
@@ -37,14 +42,28 @@ namespace flatdata
 template < typename IndexType, typename... Args >
 class MultiVector
 {
+public:
+    /// Class used to fill an individual list
+    class List
+    {
+    public:
+        template < typename T >
+        typename T::MutatorType add( );
+
+    private:
+        friend class MultiVector;
+        std::vector< unsigned char >* m_data;
+
+        List( std::vector< unsigned char >* data );
+    };
+
 public:
     using View = MultiArrayView< IndexType, Args... >;
 
     MultiVector( ExternalVector< IndexType > index, std::unique_ptr< ResourceHandle > data_handle );
 
-    template < typename T >
```

Comment:
As this change would break client code, could you please add statement to README.md with something like:

> Flatdata is under active development and our APIs might change. Please expect that when updating from one version to another.

Reply:
Done